projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/store/isa-summary.ts

```diff
@@ -59,6 +61,29 @@ export const isaGroupLabels = {
 	other: __( 'Other', 'jetpack-boost' ),
 };
 
+export function isaGroupLabel( group: keyof typeof isaGroupLabels | string ) {
```

Comment:
TIL `keyof typeof`. Btw, isn't `keyof typeof isaGroupLabels | string` equivalent of `string`. I am curious if writing it this way gives any advantage like autocompletion?

Reply:
The purpose is three-fold: 

1. It communicates with the developer reading the code that I'm not just expecting any string, I'm expecting a isaGroupLabels key.
2. IDE Autocomplete is able to pick up on that
3. When a key is stored in a variable, that variable can be just a string not an union, so this avoids unnecessary `as "foo" | "bar"` that would likely be necessary to satisfy this function call.